releasetasks/templates/tc_update_verify.yml.tmpl

```diff
@@ -0,0 +1,51 @@
+{% set uv_buildername = "{}_{}_update_verify".format(platform, branch) %}
+{% set uv_totalchunks = 12 %}
```

Comment:
total chunks is 6 currently, for buildbot. But when running it with 6 chunks on Taskcluster it took too long and timed out, I'll repeat the 6 chunks for the buildbot-bridge based jobs